dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -411,52 +412,45 @@
                                     </VisualState.Setters>
                                 </VisualState>
                                 <VisualState x:Name="Pressed">
                                     <VisualState.Setters>
                                         <Setter Target="RootGrid.Background" Value="{ThemeResource TopNavigationViewItemBackgroundPressed}" />
-                                        <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TopNavigationViewItemForegroundPressed}" />
+                                        <Setter Target="PointerRectangle.Fill" Value="{ThemeResource NavigationViewItemBackgroundPressed}" />
                                         <Setter Target="Icon.Foreground" Value="{ThemeResource TopNavigationViewItemForegroundPressed}" />
                                     </VisualState.Setters>
                                 </VisualState>
                                 <VisualState x:Name="Disabled">
                                     <VisualState.Setters>
-                                        <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource TopNavigationViewItemForegroundDisabled}" />
                                         <Setter Target="Icon.Foreground" Value="{ThemeResource TopNavigationViewItemForegroundDisabled}" />
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
-                        <StackPanel Padding="{TemplateBinding Padding}" Orientation="Horizontal">
-                            <ContentPresenter 
-                                x:Name="ContentPresenter"
-                                Content="{TemplateBinding Content}"
-                                ContentTransitions="{TemplateBinding ContentTransitions}"
-                                ContentTemplate="{TemplateBinding ContentTemplate}"
-                                Foreground="{TemplateBinding Foreground}"
-                                VerticalAlignment="Center"
-                                AutomationProperties.AccessibilityView="Raw" />
-                            <FontIcon
-                                x:Name="Icon"
-                                Margin="8,0,0,-4"
-                                VerticalAlignment="Center"
-                                AutomationProperties.AccessibilityView="Raw"
-                                FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                FontSize="8"
-                                Glyph="&#xE96E;"
-                                Foreground="{TemplateBinding Foreground}"
-                                IsHitTestVisible="False" />
-                        </StackPanel>
+                        <Rectangle 
+                            x:Name="PointerRectangle" 
```

Comment:
>PointerRectangle [](start = 36, length = 16)

Maybe name this HitTestRectangle?

Reply:
I'm up for the rename but would this be an issue for folks who are grabbing this element, for whatever reason, in their app code and depend on it? Would probably be a very rare situation....

Reply:
This is listed as a change here because I just moved the x:Name="PointerRectangle" to a new line (it was on the same line as <Rectange before) to match the common layout used in this file here (each property standalone on a single line).